ppmpy/ppm.py

```diff
@@ -1591,6 +1855,51 @@ def DsolveLgr(self, cycles1, cycles2, var='Xcld', src_func=None, src_args={}, \
              'D':1.e16*D}
         return res
 
+    def schwarzschild_bound(self, cycles, rmin=None, rmax=None):
+        '''
+        Method that finds rhe radius of the Schwarzschild boundary
+
+        Parameters:
+        cycles: list
+            List of dump numbers to be used in the analysis
+        rmin: float, optional
+            Minimum radius for the boundary search
+            If None, rmin=radin0
+        rmax: float, optional
+            Maximum radius for the boundary search
+            If None, rmax=radout0
+
+        Returns:
+        rs: float
+            Radius of the Schwarzschild boundary
+        '''
+
+        rmin_input = rmin
+        rmax_input = rmax
+
+        dump_list = any2list(cycles)
+        rs = []
+        for i, dump in enumerate(dump_list):
```

Comment:
I found it useful to plot the displacement of the SB as the simulation progresses, and I had no case so far where I had >1 SB. What about I modify it to return a list of lists so that both functionalities are there (i.e., range of dumps + all SBs)?

Reply:
When you have a series of dumps and you have a variable number of SBs as a function of dump number then the purpose of the dump-list feature is defied, and this case would hint at user error, correct? Maybe if the dump input is not a list return all SBs. If dump is list return list one SB per dump and warn if there are multiple SBs. How is that?

Reply:
I implemented this: if one dump is given, then all SBs are returned; otherwise, one SB per dump is returned.